### PR TITLE
Fix NPE in Elm path parameter

### PR DESCRIPTION
**Pull request by wing328**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh`, `./bin/openapi3/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`. If contributing template-only or documentation-only changes which will change sample output, be sure to [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) first.
- [x] Filed the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `4.1.x`, `5.0.x`. Default: `master`.
- [x] Copied the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

- fix using baseName instead of paramName
- re-enable elm in ensure-up-to-date script
- minor code formatting (java class)

To repeat the issue, run ` java -jar ./modules/openapi-generator-cli/target/openapi-generator-cli.jar generate -i modules/openapi-generator/src/test/resources/2_0/petstore-with-fake-endpoints-models-for-testing.yaml -g elm -o /tmp/openapi-generator-test-fake-petstore/2.0/elm`

cc @eriktim

### CODE REVIEW

File: modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ElmClientCodegen.java [hunk 1]
```diff
@@ -437,23 +437,23 @@ private static boolean anyOperationParam(final List<CodegenOperation> operations
         final Set<String> dependencies = new HashSet<>();
 
         for (CodegenOperation op : ops) {
-            if (ElmVersion.ELM_018.equals(elmVersion)) {
+            if (ElmVersion.ELM_018.equals(elmVersion)) { // elm 0.18
                 String path = op.path;
                 for (CodegenParameter param : op.pathParams) {
                     final String var = paramToString("params", param, false, null);
                     path = path.replace("{" + param.paramName + "}", "\" ++ " + var + " ++ \"");
```
Comment (eriktim) [inline modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ElmClientCodegen.java]: Did this change? I guess this should become `baseName` as well then.

Comment (wing328) [reply modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ElmClientCodegen.java]: Let me update that too.

Comment (wing328) [reply modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ElmClientCodegen.java]: Update via e417b421e4

**Comment by wing328** [comment]
CI failure either fixed in the master or not related to this change.

**Pull request merged**